src/api.js

```diff
@@ -2391,6 +2391,7 @@ api.declare({
   const result = worker ?
```

Comment:
probably don't do a ternary like this, it's a bit hard to read... neat, but hard to read..

Reply:
Hehe, ok. Changing this one and the ones above it to use `if` statements